acceptance/bundle/telemetry/deploy/script

```diff
@@ -1,6 +1,15 @@
 trace $CLI bundle deploy
 
-trace cat out.requests.txt | jq 'select(has("path") and .path == "/telemetry-ext") | .body.protoLogs[] | fromjson' > out.telemetry.txt
+trace cat out.requests.txt | jq 'select(has("path") and .path == "/telemetry-ext") | .body.protoLogs[] | fromjson' > telemetry.json
+
+# Note that the number 5 here is arbitrary. In practice I saw 16 mutators be recorded in acceptance test runs so 5 seems
+# like a reasonably safe number to assert that mutator execution times are being recorded.
+title "Assert that there are atleast 5 mutators for which the execution time is recorded"
+trace cat telemetry.json | jq ' .entry.databricks_cli_log.bundle_deploy_event.experimental.bundle_mutator_execution_time_ms | length > 5'
+
+# bundle_mutator_execution_time_ms can have variable number of entries depending upon the runtime of the mutators. Thus we omit it from
```

Comment:
It's only variable if it's bigger than 1000, right? Otherwise it's constant. Can we sort it and record in full?